wren-ai-service/src/web/v1/services/ask.py

```diff
@@ -39,7 +44,10 @@ class AskRequest(BaseModel):
     thread_id: Optional[str] = None
     user_id: Optional[str] = None
     history: Optional[AskHistory] = None
-    configurations: AskConfigurations = AskConfigurations(language="English")
+    configurations: Optional[AskConfigurations] = AskConfigurations(
+        language="English",
+        timezone=AskConfigurations.Timezone(name="Asia/Taipei", utc_offset="+8:00"),
+    )
```

Comment:
nitpick: i think we give the default value in class, thus skipping here can make code more clear.